controllers/gslb_controller.go

```diff
@@ -154,13 +154,15 @@ func (r *GslbReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 	// == handle delegated zone in Edge DNS
 	err = r.DNSProvider.CreateZoneDelegationForExternalDNS(gslb)
 	if err != nil {
-		return result.RequeueError(err)
+		log.Err(err).Msgf("Create zone delegation")
```

Comment:
something like this:
```
2021-04-29T17:41:31+02:00 ERR main.go:66 > Create zone delegation error="error message aaaaa"
```

Hovewer, the main purpose for error is the fact that based on error != nil,  controller-runtime decides to schedule new loop ASAP or wait for next period. In case of CreateZoneDelegationForExternalDNS we want to wait for the next period. 

Technically, we can skip logging here , because an error is already logged within `CreateZoneDelegationForExternalDNS `.